src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/Microsoft.DotNet.UpgradeAssistant.Abstractions.csproj

```diff
@@ -46,4 +46,8 @@
       <Version>5.0.0</Version>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <None Update="build\*" Pack="true" PackagePath="" />
```

Comment:
Looks like we're shipping targets files for extension authors now. We should document the targets file so extension authors know it's there and why/how they should use it.

Reply:
They shouldn't have to do anything with it. They'll be used behind the scenes to ensure the publish output is useable. I've added that to the docs.